en/firefox_privacy_notice.md

```diff
@@ -1,79 +1,85 @@
+* __Search queries__: Firefox sends search queries to your search provider to enable your search and to help you discover common phrases other people have searched for and improve your search experience (if your selected search provider supports search suggestions). Your selected search provider’s privacy notice governs the search provider’s collection and use of your data as you use the search provider’s services. [Learn more](https://support.mozilla.org/kb/search-suggestions-firefox), including how to disable this feature. Links to our default search providers' privacy policies include:
 
   * [Google](https://policies.google.com/privacy)
   * [Microsoft (Bing)](https://privacy.microsoft.com/privacystatement)
 
 Mozilla generally receives royalties when you use one of the default or optional search options in the Firefox web browser, including but not limited to Google, Bing, DuckDuckGo, and eBay. 
 
-* If you enable "Improve the Firefox Suggest Experience," we and our partners may also receive your search queries. [Learn more below](#searches). 
+__Firefox Suggest__ uses data to help deliver better information with less effort. By default, Firefox Suggest shows you recommended and sponsored content based on local data stored on your own computer, such as websites from your browsing history, bookmarks and open tabs (which is not sent to Mozilla). Firefox Suggest does send to Mozilla:
 
+* __Location data__: Firefox temporarily sends Mozilla your IP address, which we use to suggest content based on your country, state, and city. Mozilla may share location information with our [partners](https://support.mozilla.org/kb/firefox-suggest#w_who-are-mozillas-partners), but partners will not receive your IP address. In the US, Mozilla may also receive keyword location search data (such as when you search for “Boston”) and share this with our partners to provide recommended and sponsored content. Where this occurs, neither Mozilla nor our partners are able to associate the keyword search with an individual user once the search suggestion has been served.
+
+* __Technical & interaction data__: Firefox sends Mozilla data such as the number of times Firefox suggests or displays specific content and your clicks on that content, as well as basic data about your interactions with Firefox Suggest. Mozilla shares information about how many times suggestions are shown, the position of the suggestion, and suggestions clicked on with our [partners](https://support.mozilla.org/kb/firefox-suggest#w_who-are-mozillas-partners) for verification and feature improvement.
+
+If you choose to opt-in to "Improve the Firefox Suggest Experience" on your settings page, Mozilla receives and processes the following information to improve your browsing experience and to improve the service:
```

Comment:
Please use smart quote`“`, not straight quote `"`.

Reply:
I copied and pasted the “ you posted above - did that work?

Reply:
Should work! Technically one is open and one is close, but it's hard to tell most likely, ;-).